### PR TITLE
Upgrade rails to 4.2

### PR DESCRIPTION
**Pull request by scruti**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,140 +1,173 @@
 PATH
   remote: .
   specs:
-    modera (1.1.0)
+    modera (2.0.0)
```
Comment (jguitar) [inline Gemfile.lock]: `Gemfile.lock` shouldn't be in the repo

Comment (scruti) [inline Gemfile.lock]: you're right

**Comment by maxvlc** [comment]
:+1:

**Comment by gponsu** [comment]
:+1:

**Comment by d2bit** [comment]
:cat: :+1:

**Comment by maxvlc** [comment]
:+1:

**Pull request merged**